### DISCUSSION TITLE
problem trying to mount psp iso

### DISCUSSION THREAD

**Discussion by guynoirnpr** [aldostools/webMAN-MOD#603] [Q&A]
I have 4.88.2 Evilnat running on an original FAT console. Most things seem to work well except when I try to mount a PSP iso that is on NTFS. Now I see that PSP support is not listed as a feature on NTFS, but PS2, and PSX ISOs do get mounted and they are not listed as a feature either. Why I am asking here in webMAN-MOD discussion is that webMAN-Mod seems to show that it can mount a PSP ISO but if I get out of webMAN-MOD back to the XMB the ISO doesn't show up and I can't use PSPLauncher or anything else to get the ISO to run. The PSP ISO shows up on the "Refresh Web Games List" and if I click on the PSP ISO on the web page it will show that the file is mounted. It says "Game loaded successfully. Start the game using PSP Launcher." If I back out of webMAN-MOD to the XMB nothing shows up of the PSP ISO that I mounted.

**Comment by aldostools** [comment]
PSP ISO was supported from NTFS in 1.47.27:
⭐️ Support for mount PSPISO directly from NTFS (without copy to internal HDD). Requires prepISO 1.26 (or later)

Try an earlier version of webMAN MOD, just in case the feature was broken in a recent build.

I cannot verify this issue until I recover from Covid-19

**Reply by guynoirnpr** [reply]
Sorry to hear of the COVID diagnosis. I hope you are not trending towards the Hospital. Get better soon.

I will try other versions of webMAN-MOD to see if it works in earlier versions.

**Reply by aldostools** [reply]
Thank you for your good wishes. I'm stable now. I have been with fever for +5 days. I already had 2 doses of the vaccine.

Good luck getting it to work with the early version.
